### PR TITLE
Remove buildSrc

### PR DESCRIPTION
**Pull request by 3flex**
Only one function in buildSrc was actually required in the build, and moving that into build.gradle.kts was simple. Removing buildSrc will speed up build configuration.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -43,6 +46,10 @@ repositories {
 }
 
 val licenseHeaderFile = file("gradle/license-header.txt")
+fun readCopyrightHeader(licenseHeaderFile: File) =
+    licenseHeaderFile.readLines()
+        .joinToString("\n") { line -> line.substring(min(line.length, 3)) }
```
Comment (TWiStErRob) [inline build.gradle.kts:51]: Probably not in scope, but could this be simplified with one of the many `trimIndent` and friends variants?

**Review by TWiStErRob — APPROVED** [review]

**Comment by TWiStErRob** [comment]
Really nice catch 💪💡

**Comment by TWiStErRob** [comment]
Thank you for contributing these PRs! Things are getting very clean.

File: build.gradle.kts (hunk 1 above)
Comment (TWiStErRob) [reply build.gradle.kts:51]: Merging this PR, I think this could be in scope for "simplify build.gradle"

**Comment by szpak** [comment]
>  Removing buildSrc will speed up build configuration.

Definitely, especially as it is a configuration in Kotlin... Thanks.

**Pull request merged**